src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/module/CompositeModule.java

```diff
@@ -54,9 +54,9 @@
 	@ManyToMany
 	@Cascade(CascadeType.SAVE_UPDATE)
 	@JoinTable(
-			name = "MODULE_SUBMODULE",
-			joinColumns = { @JoinColumn(name = "MODULE_ID") },
-			inverseJoinColumns = { @JoinColumn(name = "SUBMODULE_ID") }
+		name = "MODULE_SUBMODULE",
```

Comment:
Why do you explicitly give a name here, but not everywhere else? Also shouldn't this be `MODULES_SUBMODULES``